src/components/crossDevice/CrossDeviceLink/style.css

```diff
@@ -111,7 +110,7 @@
   line-height: 24px;
   color: #2C3E4F;
   white-space: nowrap;
-  font-family: "Open Sans", sans-serif !important;
+  font-family: "Open Sans", sans-serif;
```

Comment:
is this font different from the rest?

Reply:
No, it's not. But for some reason on JSFiddle it was appearing as system-ui, so I had to explicitly add the font family.

Reply:
we should have a tech debt issue to track it then

Reply:
https://jira.onfido.co.uk/browse/CX-1565